portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -71,6 +75,18 @@ private void appendCommentToJson(String comment) {
       commentsJson += "\"" + comment + "\"";
   }
 
+  private void addCommentToDataStore(String comment) {
```

Comment:
Add function documentation here like 1-2 lines this will do.

Reply:
I have added function documentation, but I pushed it to the DataStoreLoad branch instead of this one.

Reply:
No problem. thanks.